internal/v1/handler.go

```diff
@@ -258,6 +258,21 @@ func (h *Handlers) GetComposeStatus(ctx echo.Context, composeId uuid.UUID) error
 	if err != nil {
 		return err
 	}
+	// Remove password from user customization
+	if composeRequest.Customizations != nil {
+		for i, u := range *composeRequest.Customizations.Users {
+			userPassword, err := u.AsUserPassword()
+			userPassword.Password = "REDACTED"
```

Comment:
Issue does specify to make it not readable again, I chose to keep the schema and just return constant indicating it was redacted, but it can be changed (e.g. to null or drop the password property?) :shrug: